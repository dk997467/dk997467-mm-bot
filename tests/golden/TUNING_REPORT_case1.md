TUNING REPORT

| verdict | max_delta_ratio | impact_cap_ratio | min_interval_ms | tail_age_ms | net_before | net_after | p95_after |
|---------|-----------------|------------------|------------------|-------------|------------|-----------|-----------|
| KEEP | 0.120000 | 0.080000 | 80.000000 | 700.000000 | 3.000000 | 2.851500 | 301.250000 |
| KEEP | 0.100000 | 0.060000 | 100.000000 | 600.000000 | 3.000000 | 2.732400 | 296.100000 |
| KEEP | 0.100000 | 0.080000 | 100.000000 | 600.000000 | 3.000000 | 2.766000 | 301.250000 |


