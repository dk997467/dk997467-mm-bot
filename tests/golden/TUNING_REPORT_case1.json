{"candidates":[{"metrics_after":{"fill_rate":0.7175,"net_bps":2.8515,"order_age_p95_ms":301.25,"replace_rate_per_min":240.0},"metrics_before":{"fill_rate":0.7,"net_bps":3.0,"order_age_p95_ms":320.0,"replace_rate_per_min":300.0},"params":{"impact_cap_ratio":0.08,"max_delta_ratio":0.12,"min_interval_ms":80.0,"tail_age_ms":700.0},"verdict":"KEEP"},{"metrics_after":{"fill_rate":0.735,"net_bps":2.7324,"order_age_p95_ms":296.1,"replace_rate_per_min":180.0},"metrics_before":{"fill_rate":0.7,"net_bps":3.0,"order_age_p95_ms":320.0,"replace_rate_per_min":300.0},"params":{"impact_cap_ratio":0.06,"max_delta_ratio":0.1,"min_interval_ms":100.0,"tail_age_ms":600.0},"verdict":"KEEP"},{"metrics_after":{"fill_rate":0.735,"net_bps":2.766,"order_age_p95_ms":301.25,"replace_rate_per_min":180.0},"metrics_before":{"fill_rate":0.7,"net_bps":3.0,"order_age_p95_ms":320.0,"replace_rate_per_min":300.0},"params":{"impact_cap_ratio":0.08,"max_delta_ratio":0.1,"min_interval_ms":100.0,"tail_age_ms":600.0},"verdict":"KEEP"}],"runtime":{"utc":"1970-01-01T00:00:00Z","version":"0.1.0"}}


